src-api/utilities/various.ts

```diff
@@ -1,3 +1,15 @@
 export const addMinutesToDate = (date: Date | string, minutes: number): Date => {
 	return new Date(new Date(date).getTime() + minutes * 60_000)
 }
+
+export const formatDate = (date: Date): string => {
+	const year = date.getFullYear()
+	const month = String(date.getMonth() + 1).padStart(2, '0')
+	const day = String(date.getDate()).padStart(2, '0')
+	const hours = String(date.getHours()).padStart(2, '0')
+	const minutes = String(date.getMinutes()).padStart(2, '0')
+
+	return `${year}-${month}-${day} ${hours}:${minutes}`
+}
```

Comment:
```js
export const formatDate = (date: Date): string =>
    new Intl.DateTimeFormat('en-GB', {
        year: 'numeric',
        month: '2-digit',
        day: '2-digit',
        hour: '2-digit',
        minute: '2-digit',
        hourCycle: 'h23',
    }).format(date).replace(',', '');
```

Reply:
In this case, since it's on backend, I prefer to have it more explicit, cuz I also have a regex for it. Intl hides the actual format.